Source/Core/HCCore.h

```diff
@@ -60,6 +60,7 @@ void HCIntegerPrint(HCInteger self, FILE* stream);
 //----------------------------------------------------------------------------------------------------------------------------------
 // MARK: - Real Operations
 //----------------------------------------------------------------------------------------------------------------------------------
+HCBoolean HCRealIsSimilar(HCReal self, HCReal other, HCReal dissimilarity);
```

Comment:
Should we renamed `dissimilarity` to `maxDissimilarity` and have use `<=`?

Reply:
I'll look into it.